basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/connections/ConnectorServicesTestCase.java

```diff
@@ -24,11 +25,13 @@
 
 import static org.junit.Assert.assertTrue;
 
+@Ignore("JBEAP-4467")
```

Comment:
As this is still not accepted as something being removed, are you sure it should be ignored?